processing/src/test/java/org/apache/druid/query/MultiValuedDimensionTest.java

```diff
@@ -161,7 +161,7 @@ public void setup() throws Exception
         "2011-01-12T00:00:00.000Z,product_1,t1\tt2\tt3,u1\tu2",
         "2011-01-13T00:00:00.000Z,product_2,t3\tt4\tt5,u3\tu4",
         "2011-01-14T00:00:00.000Z,product_3,t5\tt6\tt7,u1\tu5",
-        "2011-01-14T00:00:00.000Z,product_4,,u2"
+        "2011-01-14T00:00:00.000Z,product_4,\"\",u2"
```

Comment:
What is this change for?

Reply:
I think this is to preserve the original test, since previously csv parser would always produce empty strings here, but after this change in null compatible mode it will produce a null

Reply:
Oh I see.

Reply:
Thanks for clarifying this. I was too late 💤